lib/pdk/cli/test/unit.rb

```diff
@@ -21,6 +22,11 @@ module PDK::CLI
     run do |opts, _args, _cmd|
       require 'pdk/tests/unit'
 
+      if opts[:'puppet-version'] && opts[:'pe-version']
+        PDK.logger.error _('You can not specify both --puppet-version and --pe-version at the same time.')
+        exit 1
```

Comment:
Should this raise `PDK::CLI::ExitWithError` instead?

Reply:
Good call